x-pack/test_serverless/functional/services/index.ts

```diff
@@ -17,6 +16,7 @@ import { SvlCasesServiceProvider } from '../../api_integration/services/svl_case
 import { MachineLearningProvider } from './ml';
 import { LogsSynthtraceProvider } from './log';
 import { UISettingsServiceProvider } from './ui_settings';
+import { services as SvlApiIntegrationSvcs } from '../../api_integration/services';
```

Comment:
since it is shared by ai-integration and functional tests, lets move straight to `x-pack/test_serverless//shared/services`?

Reply:
https://github.com/elastic/kibana/pull/192216/commits/591081e6c1a3dc3949fc47c7fb25664bb31d2d02